File: Entregables/Iván Azagra Troya/ListaATxt/src/model/Persona.java
package model;

import java.util.UUID;

import static java.util.UUID.randomUUID;

public class Persona {
    private UUID id = randomUUID();
    private String user;
    private String email;

    private Persona() {}

    public Persona(String user, String email) {
        this.user = user;
        this.email = email;
    }

    public String getUser() {
        return user;
    }

    public void setUser(String user) {
        this.user = user;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    @Override
    public String toString() {
        return "Persona{" +
                "id=" + id +
                ", user='" + user + '\'' +
                ", email='" + email + '\'' +
                '}';
    }
}

File: Entregables/Azahara Blanco Rodriguez/pasar objetos a texto y a binario/src/main/kotlin/Pojo/Persona.kt
package Pojo

import java.io.Serializable
import java.util.*

class Persona(): Serializable {
    var r = Random()
    var id : String  = UUID.randomUUID().toString()
    var edad : Int = r.nextInt(80)
    var clasificacio : String = "alumno"

    constructor(id : String, edad : Int, clasidicacion : String) : this() {
        this.id = id
        this.edad = edad
        this.clasificacio =clasificacio
    }


}